### ISSUE TITLE
New release?

### ISSUE THREAD

**Issue by bhipple** [captin411/ofxclient#55]
It looks like the master branch has a few bugfixes that haven't been released since the 2.0.3 version from Apr 28, 2017.  When you get a chance, would you be willing to cut a new release?

**Comment by jantman** [comment]
I'd really appreciate a new release on PyPI as well...

**Comment by AntoniosHadji** [comment]
You can install the version in master with
```
pip install git+https://github.com/captin411/ofxclient.git@master
```

**Comment by jantman** [comment]
Yeah, that should work if you're trying to install it locally. That won't, however, work if you want to make it a dependency of another package via ``setup.py``.
